charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -129,6 +129,9 @@ controller:
 # The "maximum number of attachable volumes" per node
 volumeAttachLimit:
 
+snapshotController:
```

Comment:
You should remove this since the snapshot controller will soon no longer be installed via this chart.  Also, `snapshotController` already exists as a value.